spotify_tensorflow/dataset.py

```diff
@@ -233,6 +233,10 @@ def _append(self, v1, v2):
                         return np.vstack([v1, v2])
                     else:
                         raise ValueError("Only 1 or 2 dimensional features are supported")
+                elif type(v1) is tf.SparseTensorValue:
+                    v1_tensor = tf.SparseTensor.from_value(v1)
+                    v2_tensor = tf.SparseTensor.from_value(v2)
+                    return tf.sparse_concat(axis=0, sp_inputs=[v1_tensor, v2_tensor]).eval()
```

Comment:
Is this fine performance wise?

Reply:
Not sure, would be nice to know an alternative.  This issue is that when you eval a session you don't actually get a sparse tensor but a sparse tensor value which is only a class with 3 fields on it.  To concat correctly I can only think of this path.

We could also maybe write it ourselves but we should benchmark our python solution against the c TF one.

Reply:
This seems a bit fishy, but I might be totally overthinking this. `sparse_concat` is:

```
This op runs in `O(M log M)` time, where `M` is the total number of non-empty
values across all inputs. This is due to the need for an internal sort in
order to concatenate efficiently across an arbitrary dimension.
```

and afaiu this happens for pairs on buffer and each key for each iteration of `while self._get_buff_size() < self.batch_size:`. Yes - agree would be nice to see/do some benchmarks.

I would be fine with at least comment + issue, if you don't want to block this PR, and test on your workload.

What do you think?